app/components/TaskList.tsx

```diff
@@ -14,46 +14,14 @@ interface Props {
   projectRevision: TaskList_projectRevision$key;
 }
 
-const getStatus = (
-  forms: Array<{
-    changeStatus: string;
-    isPristine: boolean;
-    validationErrors: any[];
-  }>
-) => {
-  if (
-    forms.some((form) => form?.changeStatus === "pending" && !form.isPristine)
-  )
-    return "Incomplete";
-
-  if (
-    forms.some(
-      (form) =>
-        form.changeStatus === "staged" && form.validationErrors.length > 0
-    )
-  )
-    return <strong>Attention Required</strong>;
```

Comment:
"Attention Required" needs to be bold, so you'll have to re-add some logic for that. A tiny `TaskListStatus` React component would do the trick